go.mod

```diff
@@ -167,3 +167,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types => ./types
```

Comment:
After approval this commit must be cherry-picked, merged into main and
removed from this branch.